Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+#!groovy
+
+def python_versions = ['2.7','3.3','3.4','3.5','3.6','pypy','latest']
+def frameworks = ['django-1.8', 'django-1.9', 'django-1.10', 'django-1.11', 'django-master']
+def exclude_jobs = ['2.7_django-master', '3.3_django-1.9', '3.3_django-1.10', '3.3_django-1.11', '3.3_django-master']
+def test_jobs = [:]
+def linters = ['isort', 'flake8', 'docs']
+def linter_jobs = [:]
+
+node{
+    withEnv(["HOME=/var/lib/jenkins",
+             "PIP_CACHE=.cache/pip"
+             ]) {
+        stage('Checkout'){
+            node('linux'){
+                sh 'echo "do some SCM checkout..."'
+                checkout scm
+            }
+        }
+       
+        stage('Lint'){
```

Comment:
I assume if one stage fails, next stage is not started?

Reply:
yes afaik, I'll remove the `currentBuild` handling.